### ISSUE TITLE
Special character category labels should be localized

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#5820]
## 📝 Provide detailed reproduction steps (if any)

Currently category labels shown for special characters are not localized.

It would be nice to have translations for all other languages too.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
Duplicate reported in #12630

**Comment by mlewand** [comment]
Duplicate reported in #11440.

**Comment by wwalc** [comment]
Duplicate in #11440 as well.
